config/staging.json

```diff
@@ -4,6 +4,7 @@
     "fileName": "logs/staging.log",
     "maxSize": 1024000,
     "maxFiles": 4,
+    "console": true,
```

Comment:
This one should be `false`, no?

Reply:
No, I did it on purpose, so I could do `docker logs -f` and see the results.

Reply:
Ah, got it.